Create ui-c8y-1023-14-29-remove-length-limits-for-fragment-series-and-label-on-a-data.md (#4083)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-14-29-remove-length-limits-for-fragment-series-and-label-on-a-data.md

---
date: ""

> title: Removed length limits for fragment, series, and label fields in a data point forms

Reviewer: ```suggestion
title: Removed length limits for fragment, series, and label fields in data point forms
```

---
date: ""

> title: Remove length limits for fragment, series, and label on a data point form (#10479) [GRAFT][release/y2026] (#10678)

Reviewer 2: ```suggestion
title: Removed length limits for fragment, series, and label fields in a data point forms
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65464
version: 1023.14.29
---
# Backport

This will backport the following commits from `develop` to
`release/y2026`:
- [fix(Cockpit): [MTM-65464] Remove length limits for fragment, series,
and label on a data point form
(#10479)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/10479)

<!--- Backport version: 9.5.1 -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[MTM-65464]:
https://cumulocity.atlassian.net/browse/MTM-65464?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

> Co-authored-by: Paweł Rynarzewski <92171763+pawel-rynarzewski-c8y@users.noreply.github.com>

Reviewer 2: ```suggestion
Data point forms previously imposed character length restrictions on fragment, series, and label fields, which could prevent users from entering longer, more descriptive and detailed identifiers for their data points, or from being able to select a data point that has been created with longer identifiers via API. This change removes the length limits for the fragment, series, and label fields on data point forms, allowing users to enter values of any length without being restricted by character count constraints. Additionally, visual improvements have been made to properly handle longer labels in the data point graph or data point list.
```
